cli/beamable.common/Runtime/JsonSerializable/JsonSaveStream.cs

```diff
@@ -278,6 +278,17 @@ protected void AppendEscapedCharacter(char c)
 			}
 
 
+			public bool SerializeNestedJson(string key, ref JsonString jsonString)
+			{
+				// don't serialize the field at all if the json is null.
+				if (jsonString == null) return true; 

```

Comment:
Shouldn't be false here instead, since it couldn't serialize the thing?

Reply:
I _think_ it's that, in these cases, the (de)serialization is always valid. As in, if the JSON is `null`, the correct serialization is not writing anything so... we just don't write anything and return `true` to say "this is fine".